network/peers/subnets.go

```diff
@@ -57,7 +57,7 @@ diffLoop:
 					if len(peers) == 1 {
 						si.subnets[subnet] = make([]peer.ID, 0)
 					} else {
-						si.subnets[subnet] = peers[1:]
+						si.subnets[subnet] = peers[1:] //nolint:all
```

Comment:
it's a false positive by the linter
it says potential out of bounds, but it can't be because we're inside a loop on `peers`

Reply:
how this related to this pr lol?